### ISSUE TITLE
Do I need to train for Object part Segmentation in pointnet

### ISSUE THREAD

**Issue by binbbaz** [Daniel-Liu-c0deb0t/3D-Neural-Network-Adversarial-Attacks#5]
I have been training [pointnet ](https://github.com/charlesq34/pointnet)for the past three days as a requirement to run this code. I just finished running **_train.py_** in the home directory of [pointnet.](https://github.com/charlesq34/pointnet) Do I need to train for object part segmentation as well before moving on to pointnet2?

**Comment by Daniel-Liu-c0deb0t** [comment]
No, only classification is attacked with this code. Did you make the modifications to PointNet code as suggested in the readme?

**Comment by binbbaz** [comment]
No, I have not.
I am still facing some challenges in training pointnet2.
I will make the suggested modifications after training pointnet2

**Comment by Daniel-Liu-c0deb0t** [comment]
Might be important to note that it is probably difficult to get this code to work out-of-the-box due to weird setups I had before. I mentioned this in the readme, but a continuation is available [here](https://github.com/Daniel-Liu-c0deb0t/Adversarial-point-perturbations-on-3D-objects), which has cleaner code and more detailed setup instructions (PointNet/PointNet++ would be trained the same way). I didn't reimplement all the attack algorithms in the new code, only a couple important ones that I wanted to evaluate.

**Comment by binbbaz** [comment]
Thank you. I will be sure to check it out and ask for help when necessary.

**Comment by Daniel-Liu-c0deb0t** [comment]
I'm happy to help if you have any other questions.
